int8 num
float32 ddist

# num --> 100 = Board is boot up